var/spack/repos/builtin/packages/gdb/package.py

```diff
@@ -37,7 +37,7 @@ class Gdb(AutotoolsPackage, GNUMirrorPackage):
     version("7.11.1", sha256="57e9e9aa3172ee16aa1e9c66fef08b4393b51872cc153e3f1ffdf18a57440586")
     version("7.10.1", sha256="ff14f8050e6484508c73cbfa63731e57901478490ca1672dc0b5e2b03f6af622")
 
-    variant("python", default=True, description="Compile with Python support")
+    variant("python", default=True, description="Compile with Python support", when="@8.2:")
```

Comment:
GDB release notes say that 9.1+ is required for Windows to build with Python 3

Reply:
Is that only on Windows or on all systems? Can you share a link?

Reply:
At the moment, the GDB package builds with Autotools, so we don't yet support Windows for this package.

Reply:
https://sourceware.org/gdb/news/ | grep 9.1

Reply:
Well I'll let someone update this when they add Windows support for GDB.